pipenv/core.py

```diff
@@ -16,7 +16,7 @@
 from typing import Dict, List, Optional, Union
 
 from pipenv import environments, exceptions, pep508checker
-from pipenv._compat import decode_for_output, fix_utf8
+from pipenv._compat import fix_utf8
```

Comment:
also, no need for fix_utf8 ...

Reply:
Is it ok if I update `fix_utf8` in the whole `core.py` file? (Other functions use it in the same `core.py` file)

Reply:
No, need too. Just where you touch the code, you should replace it.

Reply:
Okay, `fix_utf8` wasn't used in the `do_check` function, so I left unchanged the other functions using `fix_utf8`.